server/api_routes/albumRoutes.js

```diff
@@ -0,0 +1,97 @@
+const chalk = require('chalk');
+const router = require('express').Router();
+const Album = require('../database/models/Album');
+const User = require('../database/models/User');
+
+// GET ALL ALBUMS FOR A GIVEN USER
+router.get('/:participantId', (req, res, next) => {
+  User.findAll({
+    where: {
+      id: req.params.participantId,
+    },
+    include: [{ model: Album, as: 'userAlbums' }],
+  })
+    .then(([user]) => {
+      console.log(chalk.green('Successfully got all albums'));
+      return res.status(200).json(user.userAlbums);
+    })
+    .catch((e) => {
+      console.error(chalk.red('Failed to find any albums', e));
+      next(e);
+    });
+});
+
+// GET A SINGLE ALBUM BY ID FOR A GIVEN USER
+router.get('/:userId/:albumId', (req, res, next) => {
+  User.findOne({
+    where: {
+      id: req.params.participantId,
+    },
+    include: [{
+      model: Album,
+      where: {
+        id: req.params.albumId,
+      },
+    }],
+  })
+    .then(([user]) => {
+      console.log(chalk.green('Successfully got album'));
+      return res.status(200).json(user.userAlbums);
+    })
+    .catch((e) => {
+      console.error(chalk.red('ERROR IN GET SINGLE album', e));
+      next(e);
+    });
+});
+
+// POST NEW ALBUM
+router.post('/', async (req, res, next) => {
+  const { album, owner } = req.body;
+  try {
+    const newAlbum = await Album.create({ name: album });
+    // owner association
+    newAlbum.setUser(owner);
+    // participant association
+    newAlbum.setUsers(owner);
+    console.log(chalk.green('Successfully CREATED album'));
+  } catch (e) {
+    console.error(chalk.red('Failed to post new album', e));
+    next(e);
+  }
+});
+
+// PUT (UPDATE) ALBUM WITH IMAGES
```

Comment:
@rrodrig003 
We will also need a PUT to update album with more participants

Reply:
added